hypervisor/arch/x86/trusty.c

```diff
@@ -372,7 +368,7 @@ static bool setup_trusty_info(struct acrn_vcpu *vcpu,
 	(void)memcpy_s(&mem->first_page.key_info, sizeof(g_key_info),
 			&g_key_info, sizeof(g_key_info));
 
-	(void)memset(mem->first_page.key_info.dseed_list, 0U,
+	(void)memset(&mem->first_page.key_info.dseed_list, 0U,
```

Comment:
I remember this needs () to split them. Check with Xiangyang

Reply:
@lifeix Hi LiFei, The precedence of operator "&" is less than operator ".", the associativity of the operator "." is left to right, so we don't need () here, thanks!

Reply:
BTW, if Operator "&" is used as Bitwise AND, we should use () for both of operand expression. thanks!